app/views/assignments/edit/_review_strategy.html.erb

```diff
@@ -24,6 +24,11 @@
     <br/>
       <%= label_tag('assignment_form[assignment][max_reviews_per_submission]', 'Maximum number of reviews per submission:') %>
       <%= text_field_tag('assignment_form[assignment][max_reviews_per_submission]', @assignment_form.assignment.max_reviews_per_submission ||= 10, size: 1) %>
+      <br/>
+      <%= label_tag('assignment_form[assignment][num_reviews]', 'Set max number of reviews done by each student') %>
+      <span id='num_reviews_per_student_threshold'>
```

Comment:
It uses num_reviews field from the assignments table. This field in assignment table is used by "instructor selected review" option. Since each assignment can have either instructor selected review option or auto selected review option, we can safely reuse the same field here.